pyproject.toml

```diff
@@ -30,6 +30,8 @@ classifiers = [
 ]
 dependencies = [
   "azure-identity>=1.22.0",
+  "azure-mgmt-authorization>=4.0.0",
+  "azure-mgmt-keyvault>=12.1.1",
```

Comment:
Maybe these two belong in the extras section? They aren't needed for people who just install the package.